cpp/src/stream_compaction/legacy/apply_boolean_mask.cu

```diff
@@ -51,6 +53,8 @@ namespace cudf {
  */
 table apply_boolean_mask(table const &input, gdf_column const &boolean_mask)
 {
+  CUDF_FUNC_RANGE();
```

Comment:
Is this necessary in the legacy source?

Reply:
No, but this helped me profile the regression! I don't think it hurts, does it?